frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -673,6 +675,16 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxWinners: Get<u32>;
 
+		/// Something that calculates the signed deposit base based on the signed submissions queue
+		/// size.
+		type SignedDepositBase: SignedDepositBase<BalanceOf<Self>>;
```

Comment:
No strong opinion but would it make sense to name the type and trait something along the lines of `DepositBaseCalculator` or something to differentiate it from two other very similar types `SignedDepositBaseIncreaseFactor` and `SignedFixedDepositBase `?

Reply:
Yes, I do agree that `SignedDepositBase` is too similar to the other `SignedDeposit*` types. I like the  `DepositBaseCalculator`, I'll update the name.

- [ ] Rename `SignedDepositBase` -> `DepositBaseCalculator`